test/e2e/suite_test.go

```diff
@@ -89,6 +93,16 @@ var _ = BeforeSuite(func(ctx SpecContext) {
 	Expect(err).NotTo(HaveOccurred())
 
 	testSuiteID = "dns-op-e2e-" + GenerateName()
+
+	geoCode := "EU"
+	if testDNSProvider == "gcp" {
+		geoCode = "europe-west1"
+	}
+	helpers.SetTestEnv("testGeoCode", geoCode)
+})
+
+var _ = AfterSuite(func() {
+	helpers.ResetTestEnvVars()
```

Comment:
Not sure there's much point in this? Would only be needed if you were actually setting environment variables that needed cleaned up.